### PR TITLE
[IntelliJ Plugin] Add minor fixes

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
This PR adds minor run/debug fixes

### CODE REVIEW

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java [hunk 1]
```diff
@@ -100,8 +108,27 @@ private VirtualFile findFile() {
         }
     }
 
+    private File searchFile(File file, String search) {
+        if (file.isDirectory()) {
+            File[] files = file.listFiles();
+            if(files!=null) {
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java]: Shall we reduce the number of nested if statements here?

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java [hunk 2]
```diff
@@ -74,16 +74,24 @@ public XSourcePosition getSourcePosition() {
...
-        VirtualFile file = LocalFileSystem.getInstance().findFileByPath(fileName);
-        if (file != null) {
-            return file;
+        Project project = myProcess.getSession().getProject();
+        String projectBasePath = project.getBaseDir().getPath();
+
+        // Removes package version.
+        String packageName = myFrame.getPackageName().split(":")[0];
+
+        // If package name is "." , file is in the ballerina project root level
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java]: Shall we add a full stop at the end.

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java]: Fixed

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java (hunk 2 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java]: Added

**Pull request merged**